constraintlayout/core/src/main/java/androidx/constraintlayout/core/ArrayRow.java

```diff
@@ -123,19 +123,19 @@ public interface ArrayRowVariables {
 
         /**
          * @TODO: add description
-         * @param var
+         * @param v
          * @param value
          * @param removeFromDefinition
          */
-        void add(SolverVariable var, float value, boolean removeFromDefinition);
+        void add(SolverVariable v, float value, boolean removeFromDefinition);
 
         /**
          * @TODO: add description
-         * @param definition
+         * @param def
          * @param removeFromDefinition
          * @return
          */
-        float use(ArrayRow definition, boolean removeFromDefinition);
+        float use(ArrayRow def, boolean removeFromDefinition);
```

Comment:
Lets keep the `definition` name. Update the implementations to use `definition` instead.

Reply:
There is another definition variable defined in this method so we cannot do that - https://github.com/androidx/constraintlayout/blob/main/constraintlayout/core/src/main/java/androidx/constraintlayout/core/SolverVariableValues.java#L442

Any thought?

Reply:
Addressed based on the comments. Sorry that I clicked on Resolve conversation on the other comment.